tests/integration/ovs_test.py

```diff
@@ -282,6 +296,15 @@ def test_create_and_remove_lag(self, port0_up, port1_up):
         assertlib.assert_absent(BRIDGE1)
         assertlib.assert_absent(BOND1)
 
+    def test_pretty_state_ovs_lag_name_first(self, eth1_up, eth2_up):
+        bridge = Bridge(BRIDGE1)
+        bridge.add_link_aggregation_port(BOND1, (ETH1, ETH2))
+
+        with bridge.create():
+            current_state = statelib.show_only((BRIDGE1,))
+            pretty_state = PrettyState(current_state)
+            assert OVS_BOND_YAML_STATE in pretty_state.yaml
```

Comment:
Since we only care about the order of keys, another/more complete possibility would be to traverse all keys and add assert its order. Then differences in the values would be ignored.

Reply:
I failed to find easy way to assert the order in yaml string output.

Current code only check the order of `name` entry which is enough.

And the unit test case already confirmed items not in priority list will be sorted.